### ISSUE TITLE
Contact page (contact.html)

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/spmssite#67]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/gbowne1/spmssite/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/gbowne1/spmssite/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/gbowne1/spmssite/issues) that already report this problem, without success.

### Current Behavior

- Navbar is messed up completely.
- Container for the Map overflows
  `<div class="container z-depth-1 my-5 px-0" style="background-color:rgb(228, 106, 218)">'
- Header container overflows 
- '<header>
  `<div class="container>' [overflow]
  ...
  
- Footer overflows @ `<footer class="footer-wrapper`

- Footer missing Instagram and YouTube social icons

- Google Maps should be a small or medium element inside the footer, not a large frame inside the main body.

- Phone Contact Field is wrong type for a phone number and should just say Phone

- Add field for Mobile Phone

- Missing the theme switch that is now on the About page (about.html) 
   (consider adding a theme.css). The theme should propagate through the rest of the site no matter what page you are on.

`Content Security Policy: Ignoring “http:” within script-src: ‘strict-dynamic’ specified` warning in the console.. which might be coming from Google Maps API.

- Footer needs Business hours 

- Remove MDBootstrap branding

Good theme colors are: 
   #0000ff blue
   #000000 black
   #808080 gray
   #ffffff white
   #c0c0c0 silver
   { maybe even brown }

 Background where possible should be a gray/silver metallic brush finish 

### Expected Behavior

- [ ] Working nav
- [ ] Correct theme

### Steps to Reproduce

Try it! :-)

**Comment by Traceydh** [comment]
Hello I would like to give this a try, can you assign it to me? :)

**Comment by gbowne1** [comment]
This is also missing an "How did you find us?" field

**Comment by Traceydh** [comment]
Ok sure I'll add a 'how did you find us' field in the form.

**Comment by gbowne1** [comment]
Thanks! :+1: :-)

**Comment by Traceydh** [comment]
Hey, this might be an obvious question but I'm kinda new to this. 
When I opened the project in chrome I can see that _navbar.scss is styling the contact page. However, looking through the folder I downloaded from github I can only see styles.css. 
From googling I understand that _navbar.scss is a SASS file (from Ruby), how do I access and edit this file? I can't make meaningful changes via only editing styles.css.

**Comment by gbowne1** [comment]
Yes, no worries at all. As for the styles, there is only styles.css

The stylesheet link in the contact.html page is `<link rel="stylesheet" href="../src/css/styles.css" />` and normally Bootstrap 5 CDN (Content Delivery Network) sends out its styles via the CDN links in the head tag, which are all scss.. this includes "_navbar.scss" and those files are not editable.

The file you should edit is styles.css in your IDE/Editor.

Thanks!

**Comment by Traceydh** [comment]
I see, that makes sense thanks!

**Comment by gbowne1** [comment]
@Traceydh 

No worries.  you're welcome.

**Comment by tejask1730** [comment]
hello i want to resolve this issue pls can you guide?

**Comment by gbowne1** [comment]
@tejask1730 are you a student or just learning?

**Comment by tejask1730** [comment]
I am Engineering student

**Comment by gbowne1** [comment]
Ok @tejask1730
